src/app/containers/MediaPlayer/helpers/metadata/index.js

```diff
@@ -1,5 +1,12 @@
 import pathOr from 'ramda/src/pathOr';
 
+const DEFAULT_IMAGE_RES = '640x360';
+
+const getThumbnailUri = aresMetadataBlock => {
+  const imageUrl = pathOr(null, ['model', 'imageUrl'], aresMetadataBlock);
+  return `https://${imageUrl.replace('$recipe', DEFAULT_IMAGE_RES)}`;
+};
```

Comment:
Should this new function be tested?

Reply:
the `mediaPlayerMetadata` is reliant on this helper and that's been tested, I believe that covers this as well

Reply:
Having looked at the previous issues relating to this ticket, why were the `mediaPlayerMetadata` tests passing when there was a problem with the output?